tsconfig-strict.json

```diff
@@ -452,6 +452,7 @@
     "core/templates/pages/profile-page/profile-page-backend-api.service.ts",
     "core/templates/pages/profile-page/profile-page-backend-api.service.spec.ts",
     "core/templates/pages/profile-page/profile-page-domain.constants.ts",
+    "core/templates/pages/profile-page/profile-page.component.ts",
```

Comment:
Please also add the spec file for this component.

Reply:
Done